readme.md

```diff
@@ -280,6 +280,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Fluxguard](https://fluxguard.com/)** - Fluxguard provides website DOM change monitoring orchestrated with Google Puppeteer, and audited by Lighthouse. Fluxguard is a freemium product, with monthly monitoring of up to 75 pages for free.
 
+* **[Foo](https://www.foo.software)** - Foo continuously captures performance audits with Lighthouse, notifies of change, and provides visualization. Users can choose to be notified via email, Slack and PagerDuty. A dashboard with charts show changes by day, week or month and provide detailed reports. Foo is a paid service available with tiers of pricing. Free trials aren't offered at this time, but a [demo dashboard](https://www.foo.software/demo/dashboard) is publicly available for viewing.
```

Comment:
```suggestion
* **[Foo](https://www.foo.software)** - Foo continuously monitors performance with Lighthouse, monitors for changes, and provides detailed visualization. Users can receive alerts via email, Slack, and PagerDuty with daily, weekly, or monthly reports. Foo is a paid service with several tiers of pricing.
```

Reply:
offered a few suggestions to trim it down to the average size here, but feel free to tweak :)

Reply:
Thanks @patrickhulce - I took your suggestion and trimmed it even further (removed duplicate `monitors` wording). Ready for re-review when you all have time. Thanks again.